### DISCUSSION TITLE
Summit 2022 Virtual / In-person / Hybrid

### DISCUSSION THREAD

**Discussion by claredillon** [InnerSourceCommons/InnerSourceMarketing#167] [General]
This discussion is to gather feedback on plans for the InnerSource Summit 2022. 

We're all conscious that in the current climate, the ability to travel is still not a given. However, we have seen some industry groups go back to having in-person events this year. Some feedback from these events is that: it's great to get back to seeing people in person; but there is often a smaller turn-out than in previous years. 

This thread is to gather feedback on the plans for InnerSource Summit 2022. The timing for our summit is likely to be the same as last year, mid Nov - as it works for various logistical reasons.

The options are:
- Fully virtual (nothing changes): this will be the default option, to have another virtual summit, like 2021. 
- Go full in-person 
- Go hybrid (there are many variations of what this might mean) - one option would be to follow FOSS Backstage example of one day virtual, one day in person. It could also mean two separate events.

Some thoughts for consideration:
- The virtual format has allowed for greater global accessibility to InnerSource Commons (c. 3x previous summit attendance).
- The physical/hybrid format will require earlier pre-planning/commitment (to start in May/June).
- The physical form will also typically require upfront investment (e.g. venue deposit).
- Please add others...

This discussion thread is to see what the community's preferences might be, to gather other views for consideration,  and to assess current thoughts on international travel (in the Nov time frame).

**Comment by gruetter** [comment]
For what it's worth, I'm currently organizing a fully hybrid conference at Bosch. It's a pain in the behind, to be honest. My preference would be to go with the FOSS backstage example: two days in person (depending on how far you have to travel, one day might not be worth it or hard to justify if you're traveling on company dime) plus one day full remote, which people on-site could still enjoy together.

**Reply by claredillon** [reply]
I am also of the opinion that hybrid is an awful lot of work to do well... I prefer to do virtual and live separately. 

It would be wonderful if we had an estimate of numbers for a physical event. For example, if approx. 50% (?) of pre-pandemic attendees are turning up to traditionally popular in-person events - can we make the same assumption for the ISC summits? And then is the overhead of planning and executing a physical summit worth it for our historical attendance?? Should/could we schedule something smaller in line with another event folks might be travelling to? Or have it standalone? Could we do something a little less formal than running the full CFP process etc.?

**Comment by MaineC** [comment]
As @gruetter mentioned the FOSS Backstage model: I'll share what exactly this model was - it may be more costly and more involved than you realize. Though I'm sure Plain Schwarz as the agency behind FOSS Backstage would be happy to help us out.

FOSS Backstage was *hybrid*.

It was one day fully remote. They used a platform that allowed for streaming talks, they did Q&A in jitsi rooms, they allowed for networking in a spatial chat.

The second day saw 1/3 of all ppl participate on-site, 2/3 of all ppl were remote. The event was located in Berlin. We had people from Berlin on-site as well as participants from north America. We had people from Berlin remote as well as participants from north America.

Travel for those who had to fly was a hassle. They came because they wanted to see their friends in person.

On day 2:
* All talks on-site were streamed in parallel to the remote platform (they hired WeCap to do that.)
* Remote talks were streamed to large screens on-site, Q&A was mediated through by moderators (WeCap again did the tech part). Giving remote speakers the option to see the audience during Q&A would have been possible as well but was deemed too much cost for this small event.
* All remote attendees had the option to post questions in chat like they did on day 1, those were handed to speakers on-site or remote through moderators.

**Comment by MaineC** [comment]
After running two events, the numbers I have seen (for close knit communities, where the goal is not reaching new audiences but coming together to see each other):

1/3 on site, 2/3 remote.

I would expect the ratio to lean much more to the remote side if the goal of the event is more outeach to new ppl.

**Comment by MaineC** [comment]
So to summarize: We need to decide what the purpose of the InnerSource summit really is.

If we want to reach new audiences and spread the word, we should have some remote offering.

If we want to make sure those people we already have here get to see each other in person, we should have some on-site offering.


What we did for FOSS Backstage: We kept the option to cancel on-site open, as rent for on-site was very low. As a first data point we asked speakers not if they would be willing to give their talk on-site, but if *the would prefer to give their talk on-site*. The split there already was roughly 1/3 to 2/3. We still had speakers change their mind closer to the event, which given the technical setup was fine.

**Comment by MaineC** [comment]
My hunch would be, that for the "meeting others in person" we might be better of, piggy-backing on another existing event. I remember that a lot of us will be traveling to Dublin in September anyway.

**Comment by claredillon** [comment]
With regard to goals - I feel that it is very important to have an event that gives the opportunity for people who are new to InnerSource to have a way to come up to speed quickly. The virtual Summit format gives that, and is globally accessible, with no-one requiring budget to participate. I love that about it. I would propose we continue to have the virtual summit, in a similar format (though perhaps with more Q&A) to last year.  The summits have proved to be an excellent way to bring more people to the community.

**Comment by claredillon** [comment]
I would also propose we have a different format event in-person. If we are all gathering to meet - we should get maximum value of having everyone in one place, and perhaps focus less on presentations (1-> many) and more on collaborative experiences. Perhaps focus on working groups to generate content or move forward a topic? With lots of opportunities to socialize too? Building strong relationships is at the core of the community, and this would be a great opportunity to do that... 

Dublin in September is a great opportunity for this - as logistics will be much easier, and most of the operations team will be local (so much less risk of travel disruption). An "InnerSource Gathering" or "InnerSource Festival" perhaps... (echoing Harvest Festivals... reaping of knowledge and celebration)!

**Reply by claredillon** [reply]
Though it might be harder for folks to get sign off on travel to a "festival"! ;)

**Reply by jsmanrique** [reply]
I ❤️ the "InnerSource Gathering" naming 😄 

On the other hand, having it in September in Dublin could be a good choice. Especially depending on how close it could be to the [Open Source Summit](https://events.linuxfoundation.org/open-source-summit-europe/). It could be a great opportunity to extend a visit to gorgeous Dubin 😉

**Reply by claredillon** [reply]
So - following up on that thread. The LF OSS Summit (with pre-conferences) runs from 12-16th - so those days are out. Would folks be willing to come over the previous week - Thur/Fri Sep 8th/9th? Best time of the year for sunshine in Ireland!!! ;)

I'm assuming we would be aiming for weekdays, rather than weekend?

**Comment by claredillon** [comment]
I'll add in here another event format that I love... Congregation (congregation.ie) is a unique event that is held annually in Cong, Galway where  the price of attendance is to write something in advance on the theme of the conference (e.g. "Leadership", "Community" etc.). Then at the event, everyone gets a turn to share what they have created (usually summarizing their blog post) with the a small group of the attendees. Everyone then discusses the topic raised for a short while and someone else takes a turn. The discussions are fabulous, the connections made are long-lasting. 

If we did something similar, it might look like this: 

- Everyone who attends brings a 10 min lightning talk. It doesn't have to be a case study, it can be a series of questions to be answered. 
- During the event, for each time slot, we break up into small groups. In each group, in each time slot, 3 people take turns to share their presentation, and there is a small group discussion after each presentation. 
- Next time slot, everyone moves to a different room, the groups are mixed again and 3 new people take a turn.

Not everyone hears everyone's talk - but everyone gets to share what they brought. It's a format that works well when everyone has something to share, and everyone has something to learn from others' perspective. 

Sharing here because I think it might fit the theme of Gathering, and the goal of sharing, and might also fit the needs of people who want to move InnerSource knowledge forward by examining the specific topics and questions closest to their hearts. . 

It's unconference-y - but we could get people to share their ideas and thoughts before they come...

**Comment by tsadler1988** [comment]
Definitely keen for some face to face time this year, but I agree this would need to be _as well as_ some sort of virtual summit to make things inclusive. Dublin gathering sounds great!

My Head of Engineering and I have been talking about hosting some sort of ISC event at BBC premises, presumably London but could be Manchester instead, if there's an appetite for it? @claredillon chatted a little today about this:
- Hackathon
  - I was thinking framed around tooling e.g. portals or metrics; perhaps providing some fake GitHub org that can be used as stub data? Keen to hear other ideas too!
  - Could tie in to a virtual summit e.g. run immediately before the summit and have some hacks demoed at the virtual summit
- Unconference or other gathering, perhaps tied into a virtual summit or perhaps completely separate 
- Consider reaching out to FINOS people as they have a big UK presence

If there's an appetite I'll find out more from my Head of Engineering as to what the options are.

**Reply by claredillon** [reply]
Through our discussion it dawned on me that we could have a number of different types of InnerSource "Gatherings" as people wanted to host in person meetings throughout the year e.g. the Congregation-style event mentioned above in the thread, or unconferences, or hackathons, or a mixture... Great to have this to consider - thanks @tsadler1988 !

**Comment by anajsana** [comment]
Just found this discussion and don't know if something has been decided already, but my impression based on past OS conferences last month is that people are returning to in-person events. The best option IMHO is always hybrid, since it allows people all around the world to engage in talks and discussions even though they are not able to travel 👍

**Reply by claredillon** [reply]
Thanks Ana - in this case, we have settled on the idea of the Gathering being in-person (with any presentation bits recorded) - but the big value prop here is going to be the working sessions. That format would be hard to share virtually - but the topics that have energy around them will be also covered in our virtual community calls. We will keep the Summit in November purely virtual for global reach. But we'll be keeping an eye on hybrid trends to see what we might want to look at for the future as we all get back into the in-person swing of things! :)
